src/averageAges.js

```diff
@@ -56,6 +75,34 @@ function calculateWomenAverageAge(people, withChildren) {
  */
 function calculateAverageAgeDiff(people, onlyWithSon) {
   // write code here
+  const womens = people.filter((item) => item.sex === 'f');
```

Comment:
```suggestion
  const womens = people.filter((person) => item.sex === 'f');
```
use understandable naming